### PR TITLE
Icons page

### PR DESCRIPTION
**Pull request by jahnvigupta**
## Fixes

Related to #63 by @annatuma

## Description

PR adds icon page to storybook

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by jahnvigupta** [comment]
@dhruvkb I have added icons page. Please have a look. But, for some reason there is no image of icon coming.

**Comment by jahnvigupta** [comment]
Screenshot from 2020-03-27 19-12-35

File: src/stories/icons.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,154 @@
+import { IconItem,IconGallery,Meta,Story,Preview } from '@storybook/addon-docs/blocks';
```
Comment (dhruvkb) [inline src/stories/icons.stories.mdx]:
```suggestion
import { IconItem, IconGallery } from '@storybook/addon-docs/blocks';
```

File: src/stories/icons.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+<Meta title="Assets/Vocabulary Icons"/>
```
Comment (dhruvkb) [inline src/stories/icons.stories.mdx]:
```suggestion
<Meta title="Assets/Vocabulary Icons" />
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Very minor code style changes

**Comment by jahnvigupta** [comment]
@dhruvkb I guess you have made the required changes.

**Review by dhruvkb — APPROVED** [review]
This is good work but there's a much simpler approach to this: embedding an `iframe` that loads the MDX story in Fonts inside the MDX story in Vocabulary.

**Comment by jahnvigupta** [comment]
> This is good work but there's a much simpler approach to this: embedding an `iframe` that loads the MDX story in Fonts inside the MDX story in Vocabulary.

Should I do it other way.

**Comment by dhruvkb** [comment]
@jahnvigupta no, don't. It's not required anymore.

**Pull request closed without merging**